src/GraphQL/Utility/FileUpload.php

```diff
@@ -281,6 +282,47 @@ public function saveFileUpload(UploadedFile $uploaded_file, array $settings): Fi
     }
   }
 
+  /**
+   * Validates an uploaded files, saves them and returns a file upload response.
+   *
+   * @param \Symfony\Component\HttpFoundation\File\UploadedFile[] $uploaded_files
+   *   The file entities to upload.
+   * @param array $settings
+   *   File settings as specified in regular file field config. Contains keys:
+   *   - file_directory: Where to upload the file
+   *   - uri_scheme: Uri scheme to upload the file to (eg public://, private://)
+   *   - file_extensions: List of valid file extensions (eg [xml, pdf])
+   *   - max_filesize: Maximum allowed size of uploaded file.
+   *
+   * @return \Drupal\graphql\GraphQL\Response\FileUploadResponse
+   *   The file upload response containing file entities or list of violations.
+   */
+  public function saveMultipleFileUploads(array $uploaded_files, array $settings): ResponseInterface {
```

Comment:
return type should be FileUploadResponse so that callers see the correct methods.